lib/view/authentication/login_screen.dart

```diff
@@ -0,0 +1,253 @@
+  final TextEditingController _password = TextEditingController();
+  final GlobalKey<FormState> _formkey = GlobalKey<FormState>();
+  @override
+  Widget build(BuildContext context) {
+    final double width = MediaQuery.of(context).size.width;
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: GestureDetector(
+        onTap: () {
+          FocusManager.instance.primaryFocus!.unfocus();
+        },
+        child: Container(
+          alignment: Alignment.center,
+          padding: EdgeInsets.symmetric(
+            horizontal: 16,
+          ),
+          decoration: BoxDecoration(
+            gradient: AppColors.bgGradient,
+          ),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+              Container(),
+              Column(
+                children: [
+                  Container(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: 0,
+                      vertical: 35,
+                    ),
+                    decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(8),
+                      color: Color(0xffffffff).withOpacity(0.05),
+                    ),
+                    child: Form(
+                      key: _formkey,
+                      child: Column(
+                        children: [
+                          HeadLine(),
+                          myInpField(
+                            label: "Phone number, username, or email",
+                            controller: _username,
+                            validate: MyValidators.v_name,
+                          ),
+                          myInpField(
+                            label: "Password",
+                            controller: _password,
+                            validate: MyValidators.v_pass,
+                            ispassword: true,
+                          ),
+                          Container(
+                            padding: EdgeInsets.symmetric(
+                              horizontal: 25,
+                            ),
+                            alignment: Alignment.centerRight,
+                            child: TextButton(
+                                onPressed: () {
+                                  // TODO: Forgot password Function call
+                                },
+                                child: Text(
+                                  "Forgot Password ?",
+                                  style: MyTextStyles.poppins500,
+                                )),
+                          ),
+                          PrimaryBtn(
+                            label: "Login",
+                            onpress: () {
+                              //  TODO: Login Function call
+                              if (_formkey.currentState!.validate()) {
+                                ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
Use customSnackbar at time of API integration

Reply:
removed the snackbar.